test/test_linalg.py

```diff
@@ -764,6 +765,7 @@ def run_test_skipped_elements(shape, batch, uplo):
             run_test_skipped_elements(shape, batch, uplo)
 
     @dtypes(torch.float32, torch.float64, torch.complex64, torch.complex128)
+    @tf32_on_and_off(1e-3)
     def test_kron(self, device, dtype):
```

Comment:
This is interesting. @zasdfgbnm, can you elaborate on the impact of tf32 on the kronecker product? @ngimel is surprised that cuBLAS tries to run a tf32 kernel for its inputs.

Reply:
kron internally uses `tensordot`, which internally uses `matmul`, which is running in tf32

Reply:
I have removed all `kron` changes in this PR. `kron` will be fixed in https://github.com/pytorch/pytorch/pull/50927, and this PR can be landed independently.